src/bp/ui-studio/src/web/typings.d.ts

```diff
@@ -34,6 +34,7 @@ declare global {
     botpress: {
       [moduleName: string]: any
     }
+    TELEMETRY_URL: string
```

Comment:
I don't see you setting this anywhere in the code, this will always be undefined. please remove this from the typing and add it only when we'll need it in the studio

Reply:
Forgot to remove after testing

Reply:
Fixed in : e135fcf